Sample Data/PublicFeeds/MITREATT&CK/MicrosoftSentinel.csv

```diff
@@ -277779,4 +277779,4 @@ N.A.,N.A.,Azure,Analytics,Microsoft Built-in Alerts,N.A.,Unusual file share acti
 N.A.,N.A.,Azure,Analytics,Microsoft Built-in Alerts,N.A.,Multiple delete VM activities,N.A.,MicrosoftCloudAppSecurity,SecurityAlert (MCAS) | McasShadowItReporting,N.A.,N.A.,N.A.,N.A.,N.A.,N.A.,https://docs.microsoft.com/cloud-app-security/investigate-anomaly-alerts,2022-05-20
 N.A.,N.A.,Azure,Analytics,Microsoft Built-in Alerts,N.A.,Ransomware activity,N.A.,MicrosoftCloudAppSecurity,SecurityAlert (MCAS) | McasShadowItReporting,N.A.,N.A.,N.A.,N.A.,N.A.,N.A.,https://docs.microsoft.com/cloud-app-security/investigate-anomaly-alerts,2022-05-20
 N.A.,N.A.,Azure,Analytics,Microsoft Built-in Alerts,N.A.,Unusual file deletion activity (by user),N.A.,MicrosoftCloudAppSecurity,SecurityAlert (MCAS) | McasShadowItReporting,N.A.,N.A.,N.A.,N.A.,N.A.,N.A.,https://docs.microsoft.com/cloud-app-security/investigate-anomaly-alerts,2022-05-20
-N.A.,N.A.,Azure,Analytics,Microsoft Built-in Alerts,N.A.,Investigation priority score increase (preview),N.A.,MicrosoftCloudAppSecurity,SecurityAlert (MCAS) | McasShadowItReporting,N.A.,N.A.,N.A.,N.A.,N.A.,N.A.,https://docs.microsoft.com/cloud-app-security/investigate-anomaly-alerts,2022-05-20
```

Comment:
This file was change by a github bot and not from my original pr. The file matches the file in master.